.github/workflows/push.yml

```diff
@@ -41,14 +41,6 @@ env:
   # S3 Repository in KAC Assets Account
   CLI_DIST_AWS_BUCKET_NAME: cli-dist-keboola-com
 
-  # GPG keys for DEB, RPM and APK Linux packages
-  DEB_KEY_PRIVATE: ${{ secrets.DEB_KEY_PRIVATE }}
-  DEB_KEY_PUBLIC: ${{ secrets.DEB_KEY_PUBLIC }}
-  RPM_KEY_PRIVATE: ${{ secrets.RPM_KEY_PRIVATE }}
-  RPM_KEY_PUBLIC: ${{ secrets.RPM_KEY_PUBLIC }}
-  APK_KEY_PRIVATE: ${{ secrets.APK_KEY_PRIVATE }}
-  APK_KEY_PUBLIC: ${{ secrets.APK_KEY_PUBLIC }}
-
```

Comment:
Tak ale jen doufám že jsem to rozdistribuoval správně. 😄 Ty private keys se jen zapisují do souborů v `Write private keys` kroku. A public keys se používají zřejmě jen při indexaci.(?)

Reply:
myslim, ze je to tak